apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java

```diff
@@ -232,6 +235,16 @@ AND id in (
             ;
             """;
 
+    private static final String TRACE_CNT_BY_WORKSPACE_ID = """
+                SELECT
+                     workspace_id,
+                     COUNT(1) as traces_count
+                 FROM traces
+                 WHERE toDate(start_time) = date_sub(DAY, 1, today())
```

Comment:
A few comments:
- Technically, you should use `toDate32` function which supports the same range as `DateTime64` type. That's they type of `start_time` and the rest of datetime fields in this table and in the application in general.
- The `start_time` is client based, so technically can be tampered. Server side timestamps should be a better choice. Probably `created_at` is the best choice at the moment.
- There's a yesterday function available in ClickHouse.

Reply:
This doesn't apply anymore due to the latest changes.